server/routes/api.js

```diff
@@ -27,6 +27,39 @@ router.get('/asset-compute-endpoint', async function(req, res) {
     });
 });
 
+
+router.get('/get-journal-url', async function(req, res) {
+    try {
+        if (!assetComputeDevTool) {
+            assetComputeDevTool = await setupAssetComputeDevTool();
+        }
+    }  catch(e) {
+        console.log(e);
+        return res.status(500).send({
+            message: 'Error setting up devtool - journal url'
```

Comment:
Maybe elaborate what is wrong with the journal url (and how it can be remediated - retrying later?).

Reply:
thanks! got rid of getJournalUrl call. Merged it with checkEventJournal. so no more enabling/disabling logic.